homeassistant/components/denonavr/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Denon AVR Network Receivers",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/denonavr",
-  "requirements": ["denonavr==0.10.12"],
+  "requirements": ["denonavr==0.11.0-dev"],
```

Comment:
Stale?

Reply:
Resolved in https://github.com/home-assistant/core/pull/85980/commits/00c4dd97ab99886f53b516b0e58ba0095f2d6694